docs/source/components/auth/a2a-auth.md

```
@@ -0,0 +1,251 @@
+- Complete Keycloak configuration instructions
+- Client and server configuration files
+- Testing and verification steps
+
+## Related Documentation
+
+- [A2A Protocol Overview](../integrations/a2a.md) - Introduction to Agent-to-Agent protocol
+- [A2A Client](../../build-workflows/a2a-client.md) - Connecting to remote A2A agents
+- [A2A Server](../../run-workflows/a2a-server.md) - Publishing NAT workflows as A2A agents
+
+
+## Examples
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)